.github/workflows/ci.yml

```diff
@@ -119,7 +119,7 @@ jobs:
                 rust-target: "x86_64-unknown-linux-gnu",
               }
     name: clippy/${{ matrix.platform.rust-target }}/${{ matrix.rust }}
-    continue-on-error: ${{ matrix.platform.rust != 'stable' }}
+    continue-on-error: ${{ matrix.rust != 'stable' }}
```

Comment:
While updating I noticed that this setting was wrong; this is why the macOS aborts in CI are not blocking pull requests.